client/index.html

```diff
@@ -17,6 +18,7 @@
 	<script>
 		var apiAddress = window.location.protocol +'//'+ window.location.hostname +':8080';
 		var dataManager = new DataManager();
+        var postManager = new PostManager(apiAddress);
 
 		$.ajaxSetup({
 			async: false
```

Comment:
It's a very bad idea that we still use `async: false`. It makes everything really slow. 

It's much easier to remove it know than later. So please remove it now and change the rest of the code so it still works in async mode (default mode).